specification/app/Microsoft.App.DynamicSessions/routes.tsp

```diff
@@ -0,0 +1,88 @@
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "./main.tsp";
+import "./models/code-execution.tsp";
+import "./models/code-execution-file.tsp";
+import "./models/common.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Microsoft.App.DynamicSessions;
+
+alias SessionTraits = NoRepeatableRequests &
+  NoConditionalRequests &
+  NoClientRequestId &
+  QueryParametersTrait<SessionIdentifier>;
+
+alias SessionOperations = Azure.Core.ResourceOperations<SessionTraits>;
+
+interface CodeExecution {
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" ""
+  @doc("Execute code in a session.")
+  @route("/execute:executeCode")
+  executeCode is Foundations.Operation<
+    {
+      ...SessionIdentifier;
+
+      @doc("The request to execute code.")
+      @body
+      codeExecutionRequest: SessionCodeExecutionRequest;
+    },
+    Foundations.AcceptedResponse<SessionCodeExecutionAsyncResponse> | Foundations.ResourceOkResponse<SessionCodeExecutionResponse>
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" ""
+  @doc("Get the code execution result.")
+  @route("/execute/{executionId}")
+  getResult is Foundations.Operation<
```

Comment:
Same, use standard operation in Azure.Core. Do not use Foundations.

Reply:
Same that not able to find a suitable standard operation, RpcOperation seems not allow path paramerters.